gluten-core/src/main/scala/io/glutenproject/execution/WholeStageTransformer.scala

```diff
@@ -50,6 +50,13 @@ case class WholeStageTransformContext(root: PlanNode, substraitContext: Substrai
 
 trait TransformSupport extends GlutenPlan {
 
+  final override def doExecute(): RDD[InternalRow] = {
+    throw new UnsupportedOperationException(
+      s"${this.getClass.getSimpleName} doesn't support doExecute")
+  }
+
+  final override lazy val supportsColumnar: Boolean = GlutenConfig.getConf.enableColumnarIterator
```

Comment:
This config is NOT applicable to velox backend, as only columnar iterator is supported. @zzcclp, @baibaichen, is this config still workable for CH backend? Thanks!

Reply:
I'm +1 to remove this config if does not break CH backend

Reply:
@loneylee, could you please help confirm? Thanks!